lib/rubocop/cop/lint/ambiguous_block_association.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Lint
+      # This cop checks for ambiguous block association with method
+      # when param passed without parentheses.
+      #
+      # @example
+      #
+      #   # bad
+      #
+      #   # It's ambiguous because there is no parentheses around `a` param
```

Comment:
I think this explanation is redundant. Also it's strange there's an empty line between `bad` and the rest of the explanation.